annotations/templates/annotations/_revisions.html

```diff
@@ -0,0 +1,22 @@
+<style>
+ summary {
+     cursor: pointer;
+ }
+ summary h4:hover {
+     text-decoration: underline;
+ }
+</style>
+
+{% if revisions %}
+    <details>
+    <summary><h4>Show change history ({{revisions|length}})</h4></summary>
```

Comment:
This appears on a separate line from the small triangle, which I find visually not so appealing :-)